drivers/classic/src/display_current_settings.c

```diff
@@ -85,12 +85,12 @@ display_current_settings(int mode)
         fprintf(LOG_DEST, "EQUAL_AREA\t\tFALSE\n");
     }
     fprintf(LOG_DEST, "RESOLUTION\t\t%f\n", global_param.resolution);
-    fprintf(LOG_DEST, "TIME_STEP\t\t%d\n", global_param.dt);
-    fprintf(LOG_DEST, "SNOW_STEP\t\t%d\n", options.SNOW_STEP);
+    fprintf(LOG_DEST, "TIME_STEP\t\t%f\n", global_param.dt);
+    fprintf(LOG_DEST, "SNOW_STEP\t\t%f\n", global_param.snow_dt);
```

Comment:
The standard is somewhat ugly. The `l` is not required in `printf`, but **is** required in `scanf`. That said, let's just do a separate commit in which we replace all `%lf` in `printf` statements with `%f`. Rather than addressing this as part of this commit, just open a separate issue for it. This is exactly the kind of cleanup we want to do going from 4.x to 5.